### PR TITLE
Add necessary bits for a new cluster profile: OpenStack NFV

### PR DESCRIPTION
**Pull request by adduarte**

### CODE REVIEW

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -70,6 +70,9 @@
     'openstack-kuryr-quota-slice': {
         'default': 2,
     },
+    'openstack-nfv-quota-slice': {
+        'default': 1,
```
Comment (EmilienM) [inline core-services/prow/02_config/generate-boskos.py:74]: I think we could use 2.

File: core-services/prow/02_config/_boskos.yaml [hunk 2]
```diff
@@ -746,6 +746,10 @@ resources:
   min-count: 2
   state: free
   type: openstack-kuryr-quota-slice
+- max-count: 1
+  min-count: 1
```
Comment (EmilienM) [inline core-services/prow/02_config/_boskos.yaml:750]: please change to 2

File: clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards/boskos.jsonnet [hunk 3]
```diff
@@ -74,4 +74,6 @@ dashboard.new(
 .addPanel(statePanel(iaas="openstack-vexxhost", displayName="OpenStack Vexxhost"), gridPos={h: 9, w: 24, x: 0, y: 0})
 .addPanel(statePanel(iaas="openstack-vh-mecha-central", displayName="OpenStack VH Mecha Central"), gridPos={h: 9, w: 24, x: 0, y: 0})
 .addPanel(statePanel(iaas="openstack-vh-mecha-az0", displayName="OpenStack VH Mecha AZ0"), gridPos={h: 9, w: 24, x: 0, y: 0})
+.addPanel(statePanel(iaas="openstack-nfv", displayName="OpenStack nfv"), gridPos={h: 9, w: 24, x: 0, y: 0})
```
Comment (EmilienM) [inline clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards/boskos.jsonnet:77]: s/nfv/NFV/

File: ci-operator/step-registry/openstack/conf/clouds/openstack-conf-clouds-ref.yaml [hunk 4]
```diff
@@ -42,6 +45,5 @@ ref:
     documentation: |-
       This will override CLUSTER_TYPE in the `commands` file if set.
   documentation: |-
-    The ipi-conf-openstack ref writes all the environmental variables required
-    to run an ipi installation of OpenShift on OpenStack to the ${SHARED_DIR}
-    directory.
+    The ipi-conf-openstack-clouds ref configures the required cloud credentials
+    and installs required  scecurity certificates.
```
Comment (EmilienM) [inline ci-operator/step-registry/openstack/conf/clouds/openstack-conf-clouds-ref.yaml:49]: some extra spaces and typos in there

File: clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards_out/boskos.json [hunk 5]
```diff
@@ -1228,6 +1228,93 @@
...
+               "format": "time_series",
+               "intervalFactor": 2,
+               "legendFormat": "{{state}}",
+               "refId": "A"
+            }
+         ],
+         "thresholds": [ ],
+         "timeFrom": null,
+         "timeShift": null,
+         "title": "OpenStack nfv Quota Leases by State",
```
Comment (EmilienM) [inline clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards_out/boskos.json:1286]: s/nfv/NFV/

**Comment by EmilienM** [comment]
/retitle Add necessary bits for a new cluster profile: OpenStack NFV

**Comment by adduarte** [comment]
lets try this again /test pj-rehearse

**Comment by EmilienM** [comment]
This is ready for review, no need to retest. Next step will be to add new jobs on that cloud.

**Comment by EmilienM** [comment]
/assign @petr-muller

Thanks

**Review by mandre — COMMENTED** [review]
/lgtm

**Review by droslean — APPROVED** [review]

**Pull request merged**